backend/src/services/demux/ActionHandler.js

```diff
@@ -1,11 +1,11 @@
-const { AbstractActionHandler } = require('demux')
-const mongoose = require('mongoose')
-const Post = require('../api/post/post.model')
-const BlockIndexState = require('../api/block-index-state/block-index-state.model')
-const io = require('../utils/io')
+import { AbstractActionHandler } from 'demux'
+import mongoose from 'mongoose'
+import { Post, BlockIndexState } from '../../models'
```

Comment:
Is NODE_PATH supported on the backend?  That allows us to keep directory references absolute and would make more terse

Reply:
I meant to do this, but i forgot, thanks for the reminder. I'll look it up now to see if it's possible

Reply:
Looks like setting NODE_PATH in .env does not work and using NODE_PATH is not a recommended practice for node anyways